data-fusion/s1_flooding_visualisation/mosaic.js

```diff
@@ -0,0 +1,34 @@
+//VERSION=3
+function setup() {
+    return {
+      input: [{
+          datasource: "CUSTOM",
+          bands: ["B02", "B03", "B04"]
+        },
+        {
+          datasource: "S1GRD",
+          bands: ["VV", "VH", "dataMask"]
```

Comment:
The VH band is not accessed, which can be removed

Reply:
done